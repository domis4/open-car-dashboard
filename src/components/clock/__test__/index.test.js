import React from 'react'
import { shallow } from 'enzyme'
import Clock from '../index'

jest.mock('moment', () => () => ({ format: () => '16:10' }))

describe('clock index.js', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<Clock />).dive()

    expect(wrapper).toMatchSnapshot()
  })
})
