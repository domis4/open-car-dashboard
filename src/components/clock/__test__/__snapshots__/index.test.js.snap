// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clock index.js expects component to match snapshot 1`] = `
<WithStyles(View)
  setTime={[Function]}
  time="16:10"
/>
`;
